internal/cli/session.go

```
@@ -87,34 +86,81 @@ func newSessionStopCmd() *cobra.Command {
 	return cmd
 }
 
-// lookupSessionID resolves a session name, ID prefix, or full ID to a full session ID.
+// lookupSessionID resolves a session name, ID prefix, or full ID to a full
+// session ID. Exact full-ID or exact name matches always win. Otherwise we
+// require exactly one prefix match — ambiguous prefixes are rejected so
+// commands do not silently address the wrong session.
 func lookupSessionID(c *Client, arg string) (string, error) {
 	sessions, err := c.ListSessions()
 	if err != nil {
 		return arg, nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 94) but it returns nil

(nilerr)